src/amo/components/AddonsCard/index.js

```diff
@@ -13,15 +13,45 @@ export default class AddonsCard extends React.Component {
     addons: PropTypes.array.isRequired,
     children: PropTypes.node,
     className: PropTypes.string,
+    loading: PropTypes.boolean,
+    // When loading, this is the number of placeholders
+    // that will be rendered.
+    placeholderTotal: PropTypes.number,
     type: PropTypes.string,
   }
 
   static defaultProps = {
+    loading: false,
+    placeholderTotal: 5,
     type: 'list',
   }
 
   render() {
-    const { addons, children, className, type, ...otherProps } = this.props;
+    const {
+      addons,
+      children,
+      className,
+      loading,
+      placeholderTotal,
+      type,
+      ...otherProps
+    } = this.props;
+
+    const searchResults = [];
+
+    if (addons && addons.length) {
+      addons.forEach((addon) => {
+        searchResults.push(
+          <SearchResult addon={addon} key={addon.slug} />
+        );
+      });
+    } else if (loading) {
+      for (let count = 0; count < placeholderTotal; count++) {
```

Comment:
It's literally only one character shorter and it's pretty obscure so I don't think it's worth it:
```js
      Array(placeholderCount).fill(0).map((value, count) => {
        searchResults.push(
          <SearchResult key={count} />
        );
      });
```

vs.

```js      
      for (let count = 0; count < placeholderCount; count++) {
        searchResults.push(
          <SearchResult key={count} />
        );
      }
```